semanticdb/semanticdb3/semanticdb3.proto

```diff
@@ -250,6 +250,7 @@ message SymbolOccurrence {
   Range range = 1;
   string symbol = 2;
   Role role = 3;
+  Type materializedTpe = 4;
```

Comment:
I suggest we call this simply `tpe`. Along the same lines, I think it would make sense to rename associated fields, methods and local variables to `tpe` (or `gtpe` / `mtpe` in DocumentOps).